src/components/ResourceList/tests/ResourceList.test.tsx

```diff
@@ -1041,7 +1074,12 @@ describe('<ResourceList />', () => {
       setSmallScreen();
       trigger(resourceList.find(EventListener), 'handler');
 
-      expect(resourceList.find(Select).first().prop('labelInline')).toBe(false);
+      expect(
```

Comment:
This sort of reformatting suggests your local prettier is outdated? Do you need to restart vscode before for it to spot updates in your node_modules?

Reply:
Huh, maybe, let me try